src/common/Error/index.js

```diff
@@ -0,0 +1,28 @@
+import { StyledNavLink } from "../Header/Navigation/styled";
+import { Wrapper, ErrorIcon, Title, Subtitle, Button } from "./styled";
+import { MainWrapper } from "../MainWrapper";
+
+export const Error = () => {
+  return (
+    <MainWrapper
+      content={
+        <Wrapper>
+          <ErrorIcon />
+          <Title>
+            Ooops! Something went wrong...
+          </Title>
+          <Subtitle>
+            Please check your network connection and try again
+          </Subtitle>
+          <StyledNavLink
+            to="/movies"
```

Comment:
This button doesn't lead anywhere, because you doesn't import { NavLink } from "react-router-dom"; by the way, you need to create a separate StyledLink for this button; please fix.